### ISSUE TITLE
Add timeframe to Predictoor Assets name

### ISSUE THREAD

**Issue by KatunaNorbert** [oceanprotocol/df-py#742]
Right now it's hard to differentiate on the df-web side which is a 1h timeframe and which is a 1d timeframe asset. They are looking exactly the same.
I propose we also add the timeframe to the asset name so we know what's the difference.

**Comment by KatunaNorbert** [comment]
I think we can also improve the overall asset name format. the 'Predictoor Asset: ' part takes a lot of space and it's very redundant and not really util. Let's make it more quicker for user to identify and differentiate assets by removing that part and make it more similar to what Predictoor website displays: `BTC-USDT 5h Binance`

**Comment by trentmc** [comment]
> `BTC-USDT 5h Binance`

Yes, but keep the pair and the exchange name side-by-side please. Therefore `BTC-USDT 5m Binance`.

(I assume you meant `5m` or `1h`)

**Comment by trentmc** [comment]
And, where precisely do you want this asset name updated? In the csvs? Update the name in the smart contract (if possible) ?

**Comment by KatunaNorbert** [comment]
The name in the contract is fine, the current name is created inside df-py csvs (https://github.com/oceanprotocol/df-py/blob/main/df_py/volume/queries.py#L358) then propagated to df-sql and then df-web.

**Comment by trentmc** [comment]
> The name in the contract is fine, the displayed name is created inside df-py csvs (https://github.com/oceanprotocol/df-py/blob/main/df_py/volume/queries.py#L358) then propagated to df-sql and then df-web. Should be easy to alternate.

Thanks, I will fix this promptly.

**Comment by trentmc** [comment]
For reference (for myself as much as anyone): the names & symbols are set in pdr-backend/publisher/publish.py, starting at [this line](https://github.com/oceanprotocol/pdr-backend/blob/f2fff230a336c81896e77f19e0e8528efecff2be/pdr_backend/publisher/publish.py#L57), as:

```python
    nft_name: str = base + "-" + quote + "-" + source + "-" + timeframe
    nft_symbol: str = pair
    erc20_name: str = nft_symbol
    erc20_symbol: str = nft_symbol
```

where `pair = base + "/" + quote`

These variables are passed into `factory.createNftWithErc20WithFixedRate()`.

The data is also available via ERC725 as fields on the data nft: (this is from the code, lightly edited for understandability)
```
    tx = data_nft.set_data("pair", pair) # eg "ETH / USDT"
    tx = data_nft.set_data("base", base) # eg "ETH"
    tx = data_nft.set_data("quote", quote) # eg "USDT"
    tx = data_nft.set_data("source", source) # eg "Binance"
    tx = data_nft.set_data("timeframe", timeframe) # eg "1h"
```

**Comment by trentmc** [comment]
Crap, I can't easily test this because I can't run barge locally easily. @trizin can you take this over the line please?

**Comment by trentmc** [comment]
And, @KatunaNorbert, there's a complementary thing that _you_ could do: make the "↗" arrow link to the data_nft's url on etherscan, that would be extremely helpful.

Details: Right now the "↗" arrow goes to predictoor.ai generally. But that's not so helpful to get extra information about the asset, because you still can't distinguish among the 20 feeds. For non-predictoor asset it goes to the asset listing in Ocean Market, with data nft address and more.
